examples/extra_panels/extra_pgsql.py

```diff
@@ -0,0 +1,100 @@
+import asyncio
+import functools
+import time
+import inspect
+
+from aiohttp_debugtoolbar.panels.base import DebugPanel
+
+from aiopg.cursor import Cursor
+
+
+__all__ = ['RequestPgDebugPanel']
+
+
+class RequestPgDebugPanel(DebugPanel):
+    """
+    A panel to display SQL queries.
+    """
+    name = 'PgSQL'
+    has_content = True
+    template = 'request_pgsql.jinja2'
+    title = 'PgSQL Queries'
+    nav_title = title
+    _original_func = None
+
+    def __init__(self, request):
+        super().__init__(request)
+
+    @asyncio.coroutine
+    def process_response(self, response):
+        self.data = data = {}
+        data.update({
+            'timing_rows': {
+                'Total time': '%0.3f sec' % self.total_time,
+                'Total': len(self._queries),
+            }.items(),
+            'queries': [(k, v) for k, v in enumerate(self._queries)],
+        })
+
+    def _wrap_handler(self, handler):
+
+        def wrapper(func):
+            @functools.wraps(func)
+            async def wrapped(*args, **kwargs):
```

Comment:
Since this is example panel, it is probably ok to leave it as it but pyflakes should check this files if python version less then 3.5